documentation/src/main/asciidoc/user_guide/integrations.adoc

```diff
@@ -1,5 +1,15 @@
 ==  Integrations
-Infinispan can be integrated with a number of other frameworks, as detailed below.
+Infinispan can be integrated with a number of other projects, as detailed below.
+
+=== Apache Spark
+
+Infinispan provides an link:$$http://spark.apache.org$$[Apache Spark] connector capable of exposing caches as an RDD, allowing batch and stream jobs to be run against data stored in Infinispan. For further details, see the link:$$https://github.com/infinispan/infinispan-spark/blob/master/README.md$$[Infinispan Spark connector documentation].
+Also check the link:$$https://github.com/infinispan/infinispan-spark/tree/master/examples/twitter/README.md$$[Docker based Twitter demo].
+
+=== Apache Hadoop
+
+The Infinispan Hadoop connector can be used to expose Infinispan as a Hadoop compliant data source and sink that implements link:$$https://hadoop.apache.org/docs/r2.7.2/api/org/apache/hadoop/mapreduce/InputFormat.html$$[InputFormat]/link:$$https://hadoop.apache.org/docs/r2.7.2/api/org/apache/hadoop/mapreduce/OutputFormat.html$$[OutputFormat].
```

Comment:
Instead of using a specific version you can use "stable" for Hadoop urls: https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/mapreduce/InputFormat.html